$OpenBSD: patch-libdialogpages_filespage_cpp,v 1.1.1.1 2019/03/03 10:52:30 rsadowski Exp $

Call QCombobox::setCurrentText to fire up the correct signals.
KUrlComboBox::setUrl blocks sending signals.


Index: libdialogpages/filespage.cpp
--- libdialogpages/filespage.cpp.orig
+++ libdialogpages/filespage.cpp
@@ -143,6 +143,7 @@ void FilesPage::doOpen(KUrlComboBox* urlComboBox, bool
     {
         // remove newUrl if already exists and add it as the first item
         urlComboBox->setUrl(newUrl);
+        urlComboBox->setCurrentText(newUrl.toString());
     }
 
 }
